src/operations.js

```diff
@@ -4,6 +4,9 @@ const operations = {}
 const mocks = {}
 
 export const register = (name, operation, mock) => {
+  if (!name) throw new Error('Unable to register: No service name has been provided')
+  if (!operation) throw new Error('Unable to register: No operation has been provided for: ' + name)
+  if (operations[name]) console.log('WARNING: Overwriting existing service for: ' + name)
```

Comment:
A future enhancement is to have some sort of environment variable or something to set logging level.  A library spitting out lots of console logs without the ability to silence them is annoying.